app/src/main/java/org/wikipedia/descriptions/DescriptionEditUtil.java

```diff
@@ -15,6 +15,11 @@
 
     public static boolean isEditAllowed(@NonNull Page page) {
         PageProperties props = page.getPageProperties();
+        if (page.getTitle().getWikiSite().languageCode().equals("en")) {
```

Comment:
Maybe it is redundant but is it a good idea to put `&& props.canEdit()` into the `if` statement?

Reply:
Technically this method tells if the *description* specifically should be editable. I'm not sure we should mix up whether the entire article is editable here. (it's already handled correctly elsewhere)